tt_metal/impl/dispatch/command_queue.cpp

```diff
@@ -126,8 +127,8 @@ const DeviceCommand EnqueueReadBufferCommand::assemble_device_command(uint32_t d
 
     // Targeting fast dispatch on remote device means commands have to be tunneled through ethernet
     // Even when targeting fast dispatch on remote device, commands are tunneled through ethernet to consumer tensix cores
-    constexpr bool cmd_consumer_on_ethernet = false;
-    uint32_t consumer_cb_num_pages = (get_consumer_data_buffer_size(cmd_consumer_on_ethernet) / padded_page_size);
+    bool cmd_consumer_on_ethernet = this->dispatch_core_type == CoreType::ETH;
```

Comment:
command_queue also had some changes after tunnelling PR. I will review this when its rebased